Newsletters/Yearn Finance Newsletter #44/Yearn Finance Newsletter #44.md

```diff
@@ -1,95 +1,95 @@
+Puedes leer una descripción detallada de las estrategias para todas nuestras yVaults activas [Aquí](https://medium.com/yearn-state-of-the-vaults/the-vaults-at-yearn-9237905ffed3).
 
-## **Ecosystem News**
+## **Noticias del ecosistema**
 
-[Longlong finance built on top of Yearn](https://twitter.com/longlongfinance/status/1424889905877069826)
+[Longlong finance ha sido construido sobre Yearn](https://twitter.com/longlongfinance/status/1424889905877069826)
 
-[Yearn’s Gnosis safe UI gets an upgrade](https://twitter.com/seanmacaonghais/status/1427229450773618695?s=21)
+[Se actualiza la interfaz de usuario de la Gnosis Safe de Yearn](https://twitter.com/seanmacaonghais/status/1427229450773618695?s=21)
 
-[Introducing Mean Finance, a DCA protocol built with Keep3r](https://twitter.com/mean_fi/status/1422947694444785666?s=21)
+[Introducción a Mean Finance, el protocolo de DCA construido con Keep3r](https://twitter.com/mean_fi/status/1422947694444785666?s=21)
 
-[Guide on using your Woofy ](https://twitter.com/cryptannews/status/1426489521911177217?s=21)
+[Guía para utilizar WOOFY](https://twitter.com/cryptannews/status/1426489521911177217?s=21)
 
-[An issue has been identified and resolved in Curve Bribe v1 by the Yearn security team](https://twitter.com/bantg/status/1426629982328180737?s=21)
+[El equipo de seguridad de Yearn ha identificado y resuelto un problema en Curve Bribe v1](https://twitter.com/bantg/status/1426629982328180737?s=21)
 
-[Thread on how Spell, Curve, and Yearn can collaborate](https://twitter.com/danielesesta/status/1426547097415913476?s=21)
+[Hilo explicando como Spell, Curve, y Yearn pueden colaborar](https://twitter.com/danielesesta/status/1426547097415913476?s=21)
 
-[Yapeswap uses Yearn to deposit idle assets when its reserves are not being utilized](https://twitter.com/yapeswap/status/1427270229839605761)
+[Yapeswap usa Yearn para depositar los activos ociosos cuando sus reservas no están siendo utilizadas](https://twitter.com/yapeswap/status/1427270229839605761)
 
-[Iron Bank, yvBOOST, TriCrypto, and the upgraded DAI vault have been added to Akropolis](https://twitter.com/akropolisio/status/1427258414229442563)
+[Iron Bank, yvBOOST, TriCrypto y la vault mejorada de DAI han sido agregadas a Akropolis](https://twitter.com/akropolisio/status/1427258414229442563)
 
-[Details on Keep3rV2 upgrade](https://twitter.com/AndreCronjeTech/status/1429021091218006023)
+[Detalles de la mejora de Keep3rV2](https://twitter.com/AndreCronjeTech/status/1429021091218006023)
 
-[Check out the story of ape.tax, as told by Facu Ameal](https://twitter.com/fameal/status/1428382076064174080?s=20)
+[Echa un vistazo a la historia de ape.tax, contada por Facu Ameal](https://twitter.com/fameal/status/1428382076064174080?s=20)
 
-[How to use Yearn V2 with Argent](https://twitter.com/argentHQ/status/1431205382865760257)
+[Como utilizar Yearn V2 con Argent](https://twitter.com/argentHQ/status/1431205382865760257)
```

Comment:
AS-IS: [Como utilizar Yearn V2 con Argent](https://twitter.com/argentHQ/status/1431205382865760257)

TO-BE: [Cómo utilizar Yearn V2 con Argent](https://twitter.com/argentHQ/status/1431205382865760257)